web/satellite/src/components/navigation/ProjectSelection.vue

```diff
@@ -225,6 +227,7 @@ export default class ProjectSelection extends Vue {
      */
     public onCreateLinkClick(): void {
         if (this.$route.name !== RouteConfig.CreateProject.name) {
+            this.analytics.pageVisit(RouteConfig.CreateProject.path);
```

Comment:
here you will be calling pageVisit for CreateProject two times if they are paid tier, and one time if they are free tier and are not taken to the "create project" screen. I think this could be fixed if you remove the line this comment is on

Reply:
Made the change.